unlock-app/src/components/interface/locks/Manage/index.tsx

```diff
@@ -7,27 +7,29 @@ import { LockDetailCard } from './elements/LockDetailCard'
 import { Members } from './elements/Members'
 import { TotalBar } from './elements/TotalBar'
 import { FiKey as KeyIcon } from 'react-icons/fi'
-import GrantKeysDrawer from '~/components/creator/members/GrantKeysDrawer'
 import { BsArrowLeft as ArrowBackIcon } from 'react-icons/bs'
 import { BiLink as LinkIcon } from 'react-icons/bi'
 import { HiOutlineShare as ShareOptionIcon } from 'react-icons/hi'
 import { useForm } from 'react-hook-form'
 import useClipboard from 'react-use-clipboard'
 import { ToastHelper } from '~/components/helpers/toast.helper'
+import { AirdropKeysDrawer } from '~/components/interface/members/airdrop/AirdropDrawer'
 
 interface ActionBarProps {
   lockAddress: string
 }
 
 const ActionBar = ({ lockAddress }: ActionBarProps) => {
   const [isOpen, setIsOpen] = useState(false)
+  const { network } = useAuth()
 
   return (
     <>
-      <GrantKeysDrawer
+      <AirdropKeysDrawer
         isOpen={isOpen}
         setIsOpen={setIsOpen}
-        lockAddresses={[lockAddress]}
+        lockAddress={lockAddress}
+        network={network!}
```

Comment:
The `network` should be the network for which `lockAddress` makes sense not (necessarily) the one the user is connected to. We should only prompt the user to switch network when they click the button _and_ they are on the wrong network.

Reply:
Yeah, I know. this is actually to not break existing code. 

The current keydrop drawer assumed network of existing user so it doesn't check for network in the URL in two places so that would need to be changed so I have a placeholder network here for now.